cpp/common/src/codingstandards/cpp/standardlibrary/String.qll

```diff
@@ -13,12 +13,12 @@ class StdBasicString extends ClassTemplateInstantiation {
   Type getAllocator() { result = getTemplateArgument(2) }
 
   /** Gets the `const charT*` type for this `basic_string` instantiation. */
-  PointerType getConstCharTPointer() {
+  PointerType getValueType() {
```

Comment:
I think we could have retained the original name, as that's how it's described in the standard. The `value_type` is defined as `charT`, so technically even if we use `getValueType()` it should be `getConstValueTypePointer()`. I'll leave it up to you if you want to change it, though - not merge-blocking.